docs/section5/Rule5-4.md

```diff
@@ -24,20 +24,20 @@
 
 - For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
 
-  - For each thermal_block from building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
 
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
 
-  - For each space in thermal zone: ```space_baseline in zone_baseline.spaces:```  
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
 
     - Get space conditioning type: ```space_conditioning_type_baseline = space_baseline.conditioning_type```  
 
-      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on space conditioning type, status type and function type: ```surface_performance_target = data_lookup(climate_zone,space_conditioning_type_baseline,"Walls, Above-Grade")```  
+      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on climate zone, space conditioning type and function type: ```surface_performance_target = data_lookup(table_G3_4,climate_zone,space_conditioning_type_baseline,"Walls, Above-Grade")```  
```

Comment:
Thanks for adding table reference

Reply:
@xingcx - it would help to split this RDS into two different ones. One could check the layers according to Appendix A (which we still need to discuss with the Schema WG) and the second can test the U factor/R value based on tables G3.X.
For the second RDS which checks the U factor/R value, you would determine the space conditioning category based on the heating and cooling capacity within the 'HeatingVentilationAirConditioning' data group. It has data elements for sensible cool capacity and sensible heat capacity.
1. First we determine is a space is conditioned. For this the sensible cool capacity >= 3.4 Btu/h-ft2. or heating capacity >= values in 90.1-2019, table 3.2.
2. If conditioned, we determine is the space is residential (we had identified residential space types for another RDS, please look that up)
3. If semiheated- an enclosed space within a building that is heated by a heating system
whose output capacity is greater than or equal to 3.4 Btu/h·ft2 of floor area but is
not a conditioned space.

Based on the SCC, the U factors need to be determined.

Reply:
@supriyagoel I think we also need to add 

> 2\. If conditioned, we determine is the space is residential (we had identified residential space types for another RDS, please look that up)

@supriyagoel I think next we need to find surfaces associated with this space that are 
(a) adjacent to ambient - these would fall under residential and non-residential columns in Table G3.4.1-8
(b) adjacent to unconditioned or semi-heated space - these would fall under semiheated column in Table G3.4.1-8
(c) all other would be unregulated and must have properties in B-RMR = P-RMR=U-RMR. (This would include interior surfaces separating conditioned spaces.)

Reply:
> 3\. If semiheated- an enclosed space within a building that is heated by a heating system
>     whose output capacity is greater than or equal to 3.4 Btu/h·ft2 of floor area but is
>     not a conditioned space.

@supriyagoel here we need to check for the following scenarios:
(a) adjacent to ambient or conditioned space - use semiheated column in Table G3.4.1-8
(b) adjacent to unconditioned or semi-heated space - B-RMR = P-RMR=U-RMR. (This would include surfaces adjacent to unconditioned or semi-heated spaces.)

Reply:
@supriyagoel another thought... the above logic uses criteria for directly conditioned spaces (heated and cooled spaces) but not for indirectly conditioned spaces. From Section 3: 
conditioned space: a cooled space, heated space, or indirectly conditioned space defined as follows:
..................................
c. indirectly conditioned space: an enclosed space within a building that is not a heated space or a cooled space, which is heated or cooled indirectly by being connected to adjacent spaces, provided
1. the product of the U-factors and surface areas of the space adjacent to connected spaces exceeds the combined sum of the product of the U-factors andsurface areas of the space adjoining the outdoors, unconditioned spaces, and to or from semiheated spaces (e.g., corridors) or
2. that air from heated or cooled spaces is intentionally transferred (naturally or mechanically) into the space at a rate exceeding 3 ach (e.g., atria).